modules/installation-aws-config-yaml.adoc

```diff
@@ -1,4 +1,4 @@
-/ Module included in the following assemblies:
+// Module included in the following assemblies:
```

Comment:
Small fix for content that should be hidden in published docs. See https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-secret-region.html#installation-aws-config-yaml_installing-aws-secret-region